final_prep/mod_zero_hero.rb

```diff
@@ -1,41 +1,65 @@
 # Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 # Declare two variables - hero_name AND special_ability - set to strings
-
+hero_name = "Batman"
+special_ability = "Billionaire crime fighter"
 # Declare two variables - greeting AND catchphrase
 #   greeting should be assigned to a string that uses interpolation to include the hero_name
 #   catchphrase should be assigned to a string that uses interpolation to include the special_ability
+greeting = "I'm Batman"
+puts "Who are you?, #{greeting}"
+catchphrase = "This city needs me!"
+puts "You don't understand, #{catchphrase}"
 
 # Declare two variables - power AND energy - set to integers
+power = 93
+energy = 99
 
 # Declare two variables - full_power AND full_energy
 #   full_power should multiply your current power by 500
 #   full_energy should add 150 to your current energy
+full_power = power * 500
+full_energy = energy + 150
 
 # Declare two variables - is_human and identity_concealed - assigned to booleans
-
+is_human = true
+identity_concealed = false
 
 # Declare two variables - arch_enemies AND sidekicks
 #   arch_enemies should be an array of at least 3 different enemy strings
 #   sidekicks should be an array of at least 3 different sidekick strings
-
+arch_enemies = ["Joker", "Penguin", "Scarecrow"]
+sidekicks = ["Robin", "Batgirl", "Alfred"]
 # Print the first sidekick to your terminal
-
+puts sidekicks[0]
 # Print the last arch_enemy to the terminal
-
+puts arch_enemies[2]
 # Write some code to add a new arch_enemy to the arch_enemies array
+arch_enemies.push "Mr. Freeze"
 
 # Print the arch_enemies array to terminal to ensure you added a new arch_enemey
-
+p arch_enemies
 # Remove the first sidekick from the sidekicks array
-
-# Print the sidekicks array to terminal to ensure you added a new sidekick
-
+sidekicks.delete_at (0)
+# Print the sidekicks array to terminal to ensure you removed a new sidekick
+p sidekicks
 # Create a function called assess_situation that takes three arguments - danger_level, save_the_day, bad_excuse
 #   - danger_level should be an integer
-#   - save_the_day should be a string a hero would say once they save the day 
+#   - save_the_day should be a string a hero would say once they save the day
 #   - bad_excuse should be a string a hero would say if they are too afraid of the danger_level
-
+danger_level = 52
+save_the_day = "You can always count on Spiderman"
+bad_excuse = "Yikes! Not today!"
+
+if danger_level >= 50
```

Comment:
I would expect to see this all wrapped up in a method. Right now, you call method assess_situation below, but I do not see that method definition.